src/components/profile-form/index.js

```diff
@@ -12,8 +12,14 @@ const Validators = {
   email: {regex: Regex.email, message: 'Please enter a valid email.'},
   password: {
     regex: Regex.password,
-    message:
-      'Please enter a password at least 8 characters long which contains only letters (upper or lowercase) and numbers.'
+    message: (
+      <div>
+        Please use at least 8 (upper or lowercase) letters, numbers, and{' '}
+        <a href="https://www.owasp.org/index.php/Password_special_characters">
```

Comment:
Embedded html might not be the most elegant way, but I wanted to link OWASP special character list to save on documentation and use a standard.